code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -168,43 +186,11 @@
 	desc = "A Russian-made revolver for drinking games. Uses .357 ammo, and has a mechanism that spins the chamber before each trigger pull."
 	origin_tech = "combat=2;materials=2"
 	mag_type = "/obj/item/ammo_box/magazine/internal/cylinder/rus357"
-	var/spun = 0
+
 
 /obj/item/weapon/gun/projectile/revolver/russian/New()
 	..()
 	Spin()

```

Comment:
This is causing runtimes by the way, `Spin()` is now a verb that expects a `usr` to be set.